src/main/java/io/github/fileanalysissuite/adaptersdk/impls/jaxrs/internal/utils/ItemMetadataFunctions.java

```diff
@@ -60,4 +60,18 @@ public static io.github.fileanalysissuite.adaptersdk.impls.jaxrs.internal.server
 
         return modelItemMetadata;
     }
+
+    private static String toRFC3339DateTimeString(final Instant time)
+    {
+        final String minRFC3339DateTime = "0000-01-01T00:00:00Z";
+        final String maxRFC3339DateTime = "9999-12-31T23:59:59Z";
+
+        if (time.isBefore(Instant.parse(minRFC3339DateTime))) {
```

Comment:
This will parse "0000-01-01T00:00:00Z" every time any date is serialized.

Reply:
@dermot-hardy 
![image](https://user-images.githubusercontent.com/93921604/201958250-b0121c2e-dc9c-4490-8623-18a235e9a941.png)
![image](https://user-images.githubusercontent.com/93921604/201958334-6d903af1-ad4c-4023-b4fe-e6f8db9acee5.png)
Not sure I understand, when I pass `Instant.Now()` it returns the correct date?

Reply:
Think about what you should do to avoid "0000-01-01T00:00:00Z", which is a `String`. needing to be parsed to an `Instant` every time any date is serialized.